### DISCUSSION TITLE
How to use it ?

### DISCUSSION THREAD

**Discussion by badboysm890** [earthboundkid/requests#60] [Support] [answered]
I'm a beginner in golang, is there anyone who could help me with how to install it as a package and use it  ,
coz I can't find any data related to it.

**Comment by kanosy88** [comment]
same 😐

**Reply by earthboundkid** [reply, marked as answer]
https://chatgpt.com/share/e/43b0f974-1436-45ff-a830-f8e14fc57126 Not vetted for accuracy.

**Reply by earthboundkid** [reply]
ChatGPT knows that requests uses method chaining but doesn't get the actual method signatures correct. Oh well, the installation stuff is correct.
